### ISSUE TITLE
Fast Execution

### ISSUE THREAD

**Issue by Kautenja** [TASVideos/fceux#15]
I have a use case where an automated system needs to process gameplay from an emulator as fast as possible. Is there a clear way to implement this kind of functionality? I've messed around with all the SDL driver code, but can't seem to get the emulator to run any fast than 60FPS.

**Comment by bbbradsmith** [comment]
I don't know how you're automating things, but if using lua there is `emu.speedmode("turbo")` in the interface, at least. If you're modifying the source directly you might look into how the turbo mode is implemented? (By default activated with the Tab key, I think.)

**Comment by Kautenja** [comment]
Indeed, I'm using a lua script that sets up the interface for the automated agent. I've been using `emu.speedmode("maximum")` which seems to net better performance than `emu.speedmode("turbo")` (around 120fps vs 45fps, with no frameskipping). My initial thought was to increase the maximum bound of the maximum speed mode from 3200% to 128000% in the source, but that seemed to have no additional effect. Perhaps overclocking the PPU would speed things up? But, I am unsure of how to accomplish this when launching fceux from a Unix shell.

**Comment by bbbradsmith** [comment]
I think there should be a configuration file that applies all the settings, so it shouldn't matter if you can't interact with the GUI after launching? (Is it still "fceux.cfg" on the SDL version?)

I don't know much about overclocking the PPU, but the Old PPU is significantly faster code vs. New PPU, at the expense of lower accuracy. Old PPU is still the default setting, I think, but just in case you might see if it's set.

**Comment by Kautenja** [comment]
ah, i do see a `fceux.cfg` in the `.fceux` directory with all the options listed. Looks like the old PPU is still in effect. Are there any other options I should check to boost emulation speed in this config file?
